### DISCUSSION TITLE
AU Core Organization Consensus Discussion

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#39] [Consensus Discussions for R1]
Check out  [AU Core Organization](http://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html), noting the draft [general conformance](http://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html) and [Must Support](http://build.fhir.org/ig/hl7au/au-fhir-core/must-support.html) data obligations.

_To see the full constraints inherited from AU Base view the Key Elements Table_

AU Core Organization should support the minimum requirements to record, search, and fetch basic information about an organisation.

Comment on:
- If this profile was mandated tomorrow, what problems would that cause for your organisation?
- What's missing?
- Should any other identifier besides HPI-O and ABN be supported in AU Core?
- Should anything change? What about the concepts you are using, is it sufficiently covered by the terminology in AU Core?

**Comment by heathfrankel** [comment]
Please allow active value of false to support cases when historical clinical resource references an Organization that is no longer active

**Reply by nielmdrec** [reply]
At Medirecords we use Organization to represent Service Providers which can be active or inactive.
So we also need support for Organization.active to be false

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587).

Motion to remove constraint on Patient.active passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/59.

**Comment by dt-r** [comment]
From @ir4y

I have a question about the fixed value for the active field in Patient, Practitioner, PractitonerRole, and Organization.
Could you please share a bit more details on why it is a fixed true value?

There are multiple use cases when you need to make a resource inactive. @heathfrankel mentioned some of them in the GitHub discussion (please check related topics for dedicated resources) and all of them make sense to me.

_Originally posted by @ir4y in https://github.com/hl7au/au-fhir-core/discussions/43_

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587).

Motion to remove constraint on Patient.active passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/59.

**Comment by heathfrankel** [comment]
When representing Hospitals, national hospital codes (assigned by DoHA I understand) are still being used in many systems. It would be good to have a consistent manner of representing these for hospitalization transfers and theatre booking scenarios.

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587).
 Outcome: close this discussion in AU Core, @heathfrankel to raise this in AU Base with more information on the source of the national hospital codes if this is still of interest.

**Comment by heathfrankel** [comment]
I think there is some value in describing the scenarios where the Organization resource should be (and should not be) used, perhaps as use cases. Specifically, how it relates to HealthcareService and Location resources and organizational structures. For example, how does Organization get used to represent primary and community care organizations with multiple clinics, shared facilities and outreach locations. Similarly how are hospital organizations represented with multiple multiple campuses, buildings departments and outpatient clinics?
Some best practice and practical examples would be beneficial to ensure implementation consistency.

**Reply by dt-r** [reply]
@heathfrankel this has been converted to issue: https://github.com/hl7au/au-fhir-core/issues/81

**Comment by heathfrankel** [comment]
Mandatory Search parameter for address maybe problematic, what is its purpose in a primary care or hospital system?
What is the purpose of searching by ABN in a primary care system where there is likely to be only one ABN?
I suggest removing the mandatory requirement to support _revinclude of provenance until we at least have a profile for it.

**Reply by dt-r** [reply]
@heathfrankel this has been converted to issue: https://github.com/hl7au/au-fhir-core/issues/82.

**Comment by dt-r** [comment]
This discussion is now closed. Feedback on AU Core should be raised using GitHub issues. Feedback might be suggesting a feature, pointing out a place where a specification is unclear or over-restrictive, identifying a spelling or grammar issue, identifying a defect, or suggesting that an entire area needs to be rethought.

Integration with HL7 JIRA is coming and when available specification feedback will be migrated to JIRA. An announcement will be posted in this discussion when it is available.

Questions on this profile can be raised in chat.fhir.org: https://chat.fhir.org/#narrow/stream/179173-australia
